### DISCUSSION TITLE
Need help please

### DISCUSSION THREAD

**Discussion by Dannysb211** [community/community#144217] [Programming Help]
Hi all;

Am after some locally ran apps, that can run on iMac 2019, Intel Core i7. Radeon Pro 555X 2 GB. 16GB Ram, for:
Video/Photo faceswap
Brighten low light videos/photos
Face recognition

I also got Bootcamp on my machine Windows 10 (if that makes any difference, but will go with the recommendation) 

Can any one advise on where to find them on GitHub and how to install them, please? it would be greatly appreciated.

Thanks ever so much.

Kind regards to all
Danny

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Dannysb211 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Dannysb211** [reply]
Hi, Thank you, am new to all this just didnt know what to do. Regards.

**Comment by denis23x** [comment]
Hi @Dannysb211 

Please, take a look here

- https://github.com/mahseema/awesome-ai-tools
- https://github.com/re50urces/Awesome-AI
- https://github.com/bigcash/awesome-ai-list-guide
- https://github.com/ai-boost/Awesome-GPTs
